### ISSUE TITLE
Rule 3: Have clear rules for making decisions

### ISSUE THREAD

**Issue by lgatto** [gvwilson/10-meeting#2]
Great article, thank you Greg.

Upon first reading, I skimmed the title of this rule (my fault), and then was confused what it was about when reading the text. Every other rule starts with a sentence that is directly related to the rule being discussed: summary (*"In charge" means ...*), examples (*If the only purpose is ...*, *If nobody cares enough ...). The first paragraph of rule 3, on the other hand, focuses on the number of participants, and then cites *Robert's Rules of Order* (which I didn't know about, probably adding to my confusion). Starting with with something along the lines of *Making decisions when many different opinions are raised can be difficult* could possibly make the reading easier. 

Hope this helps.

**Comment by cmaimone** [comment]
I agree that the intro to rule 3 could use a little work, although possibly for a different reason.  After reading it: 

> If your meeting includes more than a couple of dozen people, or if you think you need Robert's Rules of Order for any other reason, you should probably get some training in how to run and participate in meetings. 

I'm not sure how to figure out if I'm in the situation of needing Robert's Rules of Order or more training, or what more training would be, or how I would get it.  

The second sentence is directly actionable and tells me when it applies, which is very helpful.  Could the first sentence be reworked, and possibly moved, to note under what circumstances Martha's rules might not be the best choice?

As a side note, thanks for pointing me to Martha's rules!

**Comment by raynamharris** [comment]
This rule currently only focuses on how to deal with ideas and proposals that are suggested _before_ a meeting. However, there are many other types of discussion-based or community-building meetings where decision making does need to occur but with a little less rigidity. Some points on how to moderate discussion and facilitate decision making when that is not the sole purpose of the meeting would be nice.
